nrf_rpc/include/nrf_rpc.h

```diff
@@ -127,6 +136,9 @@ struct nrf_rpc_group {
 	void *ack_handler_data;
 	const char *strid;
 	nrf_rpc_err_handler_t err_handler;
+	nrf_rpc_group_bound_handler_t bound_handler;
+	bool wait_on_init;
```

Comment:
nit/suggestion: I wonder if it would make sense to have a single `flags` member instead of these two booleans (so that we can add more flags if needed without modifying the macros in the future).

Reply:
Nice suggestion. I think I will experiment with it.